semgrep-core/src/parsing/tree_sitter/Parse_csharp_tree_sitter.ml

```diff
@@ -1587,8 +1636,8 @@ and statement (env : env) (x : CST.statement) =
         | `Case_exp (v1, v2) ->
             let v1 = token env v1 (* "case" *) in
             let v2 = expression env v2 in
-            todo env (v1, v2)
-        | `Defa tok -> todo env tok (* "default" *)
+            todo_stmt env (v1, v2)
```

Comment:
There is a Default and Case in AST_generic.ml

Reply:
These are special labels, C# allows stuff like:
```
switch (n)
{
    case 1:
        cost += 25;
        break;
    case 2:
        cost += 25;
        goto case 1;
    case 3:
        goto default;
    default:
        Console.WriteLine("Invalid selection.");
}    
```
We could just generate `"__switch_case_1"` and `"__switch_default"` labels and map `goto case` and `goto default` to regular `goto`s.